apps/web/layouts/PageLayout/AuthLayout/Sidebar.tsx

```diff
@@ -0,0 +1,103 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+import {
+  ArrowLeftRightIcon,
+  DatabaseIcon,
+  DocsIcon,
+  ExportIcon,
+  ProfileIcon,
+  SupportIcon,
+} from '../../../components/icons'
+
+import type {NavLinkProps} from './NavLink'
+import {NavLink} from './NavLink'
+
+const mainNavigation: NavLinkProps[] = [
+  {
+    name: 'Connections',
+    href: '/pipelines',
```

Comment:
Shall we change this to `/connections` (to do in a follow-up PR)?

Reply:
Yea I think we should def. keep the name & href the same either way :)

Reply:
Yes, please change to `/connections` and let's keep naming consistent

Reply:
Will do in the next PR.